templates/debug.php

```diff
@@ -284,7 +288,12 @@ function stopCountdownManually() {
         <?php endforeach ?>
     </tbody>
 </table>
-<h2><?php fs_esc_html_echo_x_inline( 'SDK Versions', 'as software development kit versions', 'sdk-versions' ) ?></h2>
+<h2>
+    <button class="toggle-table" aria-expanded="true">

```

Comment:
@DanieleAlessandra I see that this is duplicated in multiple places, so I suggest storing it in a variable and then doing something like `<?php echo $toggle_button ?>`.

Also, the class should have the `fs-` prefix. It could be something like `fs-debug-table-toggle-button`. Is the `caret-icon` class needed?